### DISCUSSION TITLE
Incoming mails , auto reply with "your mail has been received, ticket number assigned = 1234 " possible?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [chaskiq/chaskiq#771] [General]
Would it be possible to 

1. Auto reply incoming emails, with a your message has been received contents, giving them a ticket ID 
2. Auto assign new replies to this ticket ID "conversation" in Chaskiq.
3. Use this like something fresh desk has..

Is this already possible at the moment, did I Overlooked it?

We want to enable conversations with customers and use Chaskiq as a central CRM / helpdesk tool to handle all incoming emails conversations and channels.

Kind regards and thanks for the best solution so far

**Comment by michelson** [comment]
Hi, yes, it's possible. You could create a plugin that reacts to new conversations. It may involve a little bit of development but it's absolutely doable.

**Comment by Jarrodsz** [comment]
Great I totally see the option as a plugin that could be written, we put it on our todo to list and lateron contribute since we really need this feature.
